homeassistant/components/broadlink/const.py

```diff
@@ -8,7 +8,7 @@
 DOMAINS_AND_TYPES = (
     (REMOTE_DOMAIN, ("RM2", "RM4")),
     (SENSOR_DOMAIN, ("A1", "RM2", "RM4")),
-    (SWITCH_DOMAIN, ("MP1", "RM2", "RM4", "SP1", "SP2")),
+    (SWITCH_DOMAIN, ("MP1", "RM2", "RM4", "SP1", "SP2", "BG1")),
```

Comment:
Move the "BG1" class to the first position, they are in alphabetical order.

Reply:
Done